funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -423,6 +424,14 @@ def quiesce(self):
         self._quiesce_event.set()
         self._task_puller_thread.join()
         self._command_thread.join()
+
+        logger.info("Saving unacked results to disk")
+        try:
+            self.results_ack_handler.persist()
+        except Exception:
+            logger.exception("Caught exception while saving unacked results")
+            logger.warning("Interchange will continue without saving unacked results")
```

Comment:
I think maybe just a try block with logging makes sense. It is unlikely that the persist will fail since we ensure it works before initial start of the endpoint. More importantly, we don't want the endpoint to shut down simply because of this non-critical piece failing (It is bad that unacked results are not saved, but it is worse to let the endpoint completely stop unexpectedly)

Reply:
If the write to disk fails, there's little else we could do here. Logging and continuing makes sense.